examples/zh/remoteBuild.md

```diff
@@ -9,7 +9,7 @@
 <!-- @remoteOverlayBuild @test -->
 
 ```bash
-target="github.com/kubernetes-sigs/kustomize/examples/multibases/dev/?ref=v1.0.6"
+target="https://github.com/kubernetes-sigs/kustomize//examples/multibases/dev/?ref=v1.0.6"
```

Comment:
This file is technically also outdated but I can't write Chinese so I just updated the URLs to conform to the "supported" format. Someone will have to translate it if necessary.

Reply:
I can't help with that but maybe someone will come along to fix it later :)